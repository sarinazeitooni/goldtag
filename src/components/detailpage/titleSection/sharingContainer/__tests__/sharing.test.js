import React from 'react';
import { shallow } from 'enzyme';
import SharingContainer from "../sharing";
describe("SharingContainer component" , ()=>{
    it("it should render a div",()=>{
        const container = shallow(<SharingContainer titleSectionData={''}/>)
        expect(container).toMatchSnapshot();
    });
});