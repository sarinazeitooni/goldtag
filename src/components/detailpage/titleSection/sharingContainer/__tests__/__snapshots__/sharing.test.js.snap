// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SharingContainer component it should render a div 1`] = `ShallowWrapper {}`;
